automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -82,7 +87,15 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
                    expectedStatuses: Iterable[ClusterStatus],
```

Comment:
`Status` or `Statuses`?

Reply:
surprise to me, googled this. `status and statuses can both be used as the plural, depending on the context`

Reply:
I've heard `stati` too.
I personally use `statuses`, others may disagree. :)

Reply:
etymology of status is Latin, so stati is plausible, except that we're not dead Romans so their grammar rules are irrelevant to us